src/test/java/org/springframework/cloud/dataflow/acceptance/test/TickTockTests.java

```diff
@@ -57,6 +58,7 @@ public void tickTockTestsFromConfigServer() {
 				.definition("time | log")
 				.addProperty("app.log.spring.profiles.active", "test")
 				.addProperty("deployer.log.cloudfoundry.services", "cloud-config-server")
+				.addProperty("app.log.spring.cloud.config.name", "scdf-server-TICKTOCK-config-server-log")
```

Comment:
I believe we need to fix this name when this test runs on Skipper mode as the name would include version (v1 in this case)